pkg/webhook/certificate_builder.go

```diff
@@ -87,17 +92,57 @@ func (c *CertificateBuilder) genCA(ns, svc, certPath string) (*generator.Artifac
 	return certs, nil
 }
 
-// PatchCABundle patch the caBundle to MutatingWebhookConfiguration
-func (c *CertificateBuilder) PatchCABundle(webHookName string, ca []byte) error {
+// PatchCABundle watch the MutatingWebhookConfiguration and keep patching it
+func (c *CertificateBuilder) PatchCABundle(webhookName string, ca []byte) error {
+
+	err := c.PatchCABundleOnce(webhookName, ca)
+	if err != nil {
+		return err
+	}
+
+	go func() {
```

Comment:
I suggest leveraging controller runtime framework. Can we try to avoid using client-go informer directly?
Like https://github.com/fluid-cloudnative/fluid/blob/master/pkg/ctrl/watch/manager.go#L51:6

Reply:
> I suggest leveraging controller runtime framework. Can we try to avoid using client-go informer directly?
> Like https://github.com/fluid-cloudnative/fluid/blob/master/pkg/ctrl/watch/manager.go#L51:6

Thanks, I'll try to learn it.